### ISSUE TITLE
Does the Quickstart / Installation Page Work on a new Ubuntu 16.04 VM?

### ISSUE THREAD

**Issue by ttmc** [bigchaindb/bigchaindb-driver#315]
The [current Quickstart / Installation page](https://docs.bigchaindb.com/projects/py-driver/en/v0.4.0/quickstart.html) for the BigchainDB Python Driver only has one OS-level dependency installation:
```text
sudo apt-get update
sudo apt-get install libffi-dev
```
in Step 1. That really doesn't seem like enough, because the `bigchaindb_driver` package depends on the `bigchaindb` package, which needs a lot more OS-level dependencies, i.e. it needs:
```text
sudo apt-get install g++ python3-dev libffi-dev build-essential libssl-dev
```

We should _not_ assume they already have `bigchaindb` (BigchainDB Server) installed. Most people will never run BigchainDB Server.

## To Do

* Test the existing Quickstart / Installation instructions for the Python Driver on a new Ubuntu 16.04 VM.
* Add any needed / missing OS-level dependencies to the Quickstart / Installation page. Re-name that step as "Step 1: OS-level dependencies". It doesn't have to go into the details of what each one is for. You can even cut the existing explanation of the libffi-dev package. It's too much information.

**Comment by muawiakh** [comment]
fixes #316

**Comment by muawiakh** [comment]
No, it does not. Requested a PR
